src/Entity/RatioHistory.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tbbc\MoneyBundle\Entity;
+
+use DateTime;
+use DateTimeInterface;
+
+/**
+ * Class RatioHistory.
+ */
+class RatioHistory
+{
+    protected mixed $id = null;
```

Comment:
Not sure if this needs to be mixed. Maybe ?int instead?

Reply:
I agree - though it should actually be protected, so it can be used as a "super entity" and overwritten with fx a UUID field, so yea it should be mixed, because it can be both a int, string (maybe other obscure identify fields)

Reply:
Im using a UUID for it, which is a UUID interface, which is added as a string.